.dockerignore

```diff
@@ -22,6 +22,11 @@
 # build
 !samples/apps/video-streaming-app
 
+# The streaming app is built using a `docker build` command, so
+# the source code needs to be available to the container
+# build
+!discovery-handler-modules/*
```

Comment:
is this needed?  i see the discovery handlers below in the target folder ... are they built in the crossbuild?

Reply:
good point! will fix

Reply:
fixed in 5e32fd2